jarviscli/plugins/market_buy.py

```diff
@@ -68,6 +49,15 @@ def __call__(self, jarvis, s):
             else:
                 for f in files:
                     if f == 'requirements.txt':
-                        os.system('env/bin/pip install -U -r ' + os.path.join(root, f))
+                        jarvis_folder = os.path.split(self.MARKETPLACE_PATH)[0]
+                        os.system((f'cd {jarvis_folder} && ' +
+                                   'env/bin/pip install -U -r ' +
+                                   os.path.join(root, f)))
+
+        # Deleting the .git folder from the cloned repo.
+        shutil.rmtree(os.path.join(plugin_dir, '.git'))
+
+        if self.MAINTAIN_PYTHON_ONLY:
```

Comment:
Actually, don't see a benefit; something files like LICENSE or README don't waste any significant space and are nothing I would delete.